.github/workflows/tests.yml

```diff
@@ -27,20 +39,32 @@ jobs:
         pip install "poetry==1.0.10"
         poetry config virtualenvs.create false && poetry install --no-interaction --no-ansi
 
-    - name: Linters
-      if: matrix.os == 'ubuntu-latest' && matrix.python-version == '3.6'
-      run: |
-        flake8 .
-        mypy .
```

Comment:
ah yes I missed that one, thats a pain.
would it not be better to add mypy to pre-commit config as a local hook?
otherwise I think it should be part of either the pre-commit job, or just a seperate job, rather than mixed in to the pytest runs

Reply:
> would it not be better to add mypy to pre-commit config as a local hook?

I think I would maybe prefer not to, in order to keep the hooks completely independent of any external envs.

> otherwise I think it should be part of either the pre-commit job, or just a seperate job, rather than mixed in to the pytest runs

You are probably right. I would vote for separate job.

Reply:
yeh the local hooks are a pain as obviously you can't install pre-commit, there I use tox to run pre-commit.
But yeh I will add a seperate job, and also add the running of mypy to tox